nms/app/views/alarms/components/alertmanager/Receivers/Receivers.tsx

```diff
@@ -160,19 +218,27 @@ export default function Receivers() {
 
   return (
     <Grid className={classes.root} container spacing={2} direction="column">
-      {alertManagerGlobalConfigEnabled === true && (
-        <Grid item>
-          <Button
-            onClick={() => setIsEditGlobalSettings(true)}
-            startIcon={<SettingsIcon />}
-            variant="outlined">
-            Settings
-          </Button>
-        </Grid>
-      )}
       <Grid item>
         <>
+          <CardTitleRow
+            label="Alert rules"
+            icon={GroupIcon}
+            filter={() => (
+              <ReceiverFilter
+                showGlobalConfig={alertManagerGlobalConfigEnabled ?? false}
+                setShowGlobalSettings={() => setIsEditGlobalSettings(true)}
+                onAddReceiverClick={() => openAddReceiver()}
+              />
+            )}
+          />
           <SimpleTable
+            localization={{
```

Comment:
Looks like there is some typescript error in this line.

Reply:
oops yes this PR should be rebased on my other open PR #13582